File: 1) PYTHON/idade.py
print( "Dados da primeira pessoa: " )
nome1 = input( "Digite o nome da primeira pessoa: " )
idade1 = int(input( "Digite a primeira idade: " ))

print( "Dados da segunda pessoa: " )
nome2 = input( "Digite o segundo nome: " )
idade2 = int(input( "Digite a segunda idade: " ))
media = ( idade1 + idade2 ) / 2

print( f"A média das idades de, { nome1 } e { nome2 } é de: { media:.1f } anos" )

File: 6) C E C++/idade.c
#include <stdio.h>

int main( ) {
	int idade1, idade2, media;
	char nome1[ 15 ], nome2[ 15 ];
	
	printf( "Digite o primeiro nome: " );
	scanf( "%14s", nome1 );
	printf( "Digite a primeira idade: " );
	scanf( "%d", &idade1 );
	
	printf( "Digite o segundo nome: " );
	scanf( "%14s", nome2 );
	printf( "Digite a segunda idade: " );
	scanf( "%d", &idade2 );
	
	media = ( idade1 + idade2 ) / 2;

	printf( "Media: %d\n", media );
}